pimd/pim_upstream.c

```diff
@@ -1997,6 +1997,9 @@ static bool pim_upstream_kat_start_ok(struct pim_upstream *up)
 		return false;
 
 	pim_ifp = ifp->info;
+	if (!pim_ifp)
```

Comment:
We should not call this function when pim is already disabled. The fix is not looking good to me